src/YesSql.Core/Sql/BaseComandInterpreter.cs

```diff
@@ -312,5 +312,24 @@ private void Run(StringBuilder builder, ICreateColumnCommand command)
                 builder.Append(" unique");
             }
         }
+
+        /// <summary>
+        /// When the given column name contains length (ex, Alias(50)), we return the actual column name
+        /// which is in this example 'Alias'.
+        /// Some providers like MySQL support specifying length of each column in a combined index.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        protected virtual string GetRawColumnName(string name)
```

Comment:
I would prefer to see the `Run` method rewritten using a local function. `GetRawColumnName` doesn't make sense on the base class. And the name doesn't even match what it's used for.

Reply:
@sebastienros I override the run function. Please approve the PR Pipeline to run so we can see the result.